### ISSUE TITLE
Code snippet for LSTM NN 

### ISSUE THREAD

**Issue by nicola-orlando** [nicola-orlando/covid-19-time-series#1]
model = Sequential()                                                                                                                                                                                    
model.add(LSTM(50))                                                                                                                                                                                    
model.add(Dense(1))                                                                                                                                                                                    
model.compile(loss='mae', optimizer='adam')                                                                                                                                                            

                                                                                                                                                                                          
print('Trying to define the model')                                                                                                                                                                    
values_train_X = X_train.values                                                                                                                                                                        
values_train_y = y_train.values                                                                                                                                                                        
values_train_X = values_train_X.reshape((values_train_X.shape[0], 1, values_train_X.shape[1]))                                                                                                         
print(values_train_X)                                                                                                                                                                                  
print(values_train_y)                                                                                                                                                                                  
print('Trying to fit the model')                                                                                                                                                                       batch_size=int(len(values_train_y)/2)                                                                                                                                                                  
history = model.fit(values_train_X, values_train_y, epochs=1000, batch_size=batch_size, verbose=2, shuffle=False)                                                                                      
plt.plot(history.history['loss'], label='train')                                                                                                                                                       
plt.legend()                                                                                                                                                                                           
plt.show()                                                                                                                                                                                             
plt.savefig('model.png')                                                                                                                                                                               
plt.close()                                                                                                                                                                                            

y_pred = model.prediction(values_train_X)                                                                                                                                                              
history = model.fit(values_train_X, values_train_y)`
